charts/internal/shoot-crds/charts/volumesnapshots/templates/crd-volumesnapshotclasses.yaml

```
@@ -3,8 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
-    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/419"
+    controller-gen.kubebuilder.io/version: v0.8.0
```

The keep-object annotation has also gone lost by my PR #487 :-(
Let's reintroduce it

```suggestion
    resources.gardener.cloud/keep-object: "true"
    controller-gen.kubebuilder.io/version: v0.8.0
```